LICENSE.txt

```diff
@@ -652,7 +652,7 @@ Also add information on how to contact you by electronic and paper mail.
   If the program does terminal interaction, make it output a short
 notice like this when it starts in an interactive mode:
 
-    {{project}}  Copyright (C) {{year}}  {{fullname}}
+    Copyright (C) <year> <name of author>
```

Comment:
I'll just close this pr and redo the whole thing based off the standard to make sure we get it all.